.github/workflows/ci.yml

```diff
@@ -38,12 +39,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - { rust: stable, os: ubuntu-latest }
-          - { rust: stable, os: windows-latest }
-          - { rust: stable, os: macos-latest }
-          - { rust: beta, os: ubuntu-latest }
-          # Turn this on once we have a MSRV (minimum supported rust version)
-          # - { rust: 1.70.0, os: ubuntu-latest }
+          - { rust: 1.85.0, os: ubuntu-latest }
+          - { rust: 1.85.0, os: windows-latest }
+          - { rust: 1.85.0, os: macos-latest }
```

Comment:
Do you want to include a `stable` build just for checking the latest version, even if it's only against one of the platforms?

Reply:
I don't think it's necessary. Rust promises backward compatibility, but they don't promise that new versions don't introduce new clippy warnings. That can lead to builds failing and blocking PRs for reasons unrelated to the new code, which is just annoying.

I do need to change this to 1.85.1 though. That patch release only just came out.